app/test_main.py

```diff
@@ -1 +1,41 @@
-# write your code here
+from unittest import mock
+
+import pytest
+
+from app.main import can_access_google_page
+
+
+@pytest.mark.parametrize(
+    "is_connected,is_valid_url,expected_result",
+    [
+        pytest.param(
+            True, True, "Accessible",
+            id="valid connection, valid url"
+        ),
+        pytest.param(
+            False, True, "Not accessible",
+            id="invalid connection"
+        ),
+        pytest.param(
+            True, False, "Not accessible",
+            id="invalid url"
+        ),
+        pytest.param(
+            False, False, "Not accessible",
+            id="invalid connection and invalid url"
+        ),
+    ]
+)
+def test_can_access_google_page(
+        is_connected: mock,
```

Comment:
Wong type annotations for is_connected and is_valid_url. They are supposed to be boolean values.